src/main/java/com/greenfoxacademy/goddesstribesbackend/services/KingdomService.java

```diff
@@ -35,6 +41,22 @@ public Kingdom saveKingdom(String kingdomName, User user) {
     return null;
   }
 
+  public void initKingdom(String username) {
+    if (username != null && userService.checkUserByName(username)) {
+      Kingdom kingdom = kingdomRepository.findKingdomByUser_Username(username).get();
```

Comment:
Let's be consistent with where do we do the null checks! I think they do belong to the Controller instead of the service.

Reply:
(This is arguable, however do not do it in both.)